lib/validation.tsx

```diff
@@ -206,8 +206,21 @@ export const getErrorList = (
 ): JSX.Element | null => {
   let errorList;
   const formElementErrors = Object.entries(props.errors);
-  if (props.touched && formElementErrors.length) {
-    errorList = formElementErrors.map(([formElementKey, formElementErrorValue]) => {
+  if (formElementErrors.length === 0) {
+    return null;
+  }
+  if (props.children === undefined) {
+    return null;
+  }
+  const formElements = Object.entries(props.children as Array<unknown>);
```

Comment:
I'm not sure this is the best way of doing this. Is there any reason we are not using the layout array in the form config? The formConfig should be available in the props

Reply:
If you get the layout array you could do a map on it and return the relevant error if it exists

Reply:
> I'm not sure this is the best way of doing this. Is there any reason we are not using the layout array in the form config? The formConfig should be available in the props

I was hoping that the formConfig would have been available in the props. However, it doesn't seem to be available as an `InnerFormProps` or `FormikProps`. The only way I'd be able to access it is by refactoring some components higher up in the chain, and then passing in the layout. I figured that since the `children` prop was available, that could just as easily be used.

Reply:
We can discuss tomorrow. Im fairly certain its available but I could totally be wrong